README.md

```diff
@@ -107,9 +107,16 @@ Here is an examples of toplevel OCaml code:
 
 ### File sync
 `mdx` is also capable of including content from files in fenced code blocks
-using the label `file`. When an OCaml file is included it can be automatically
-sliced if it contains annotations such as `[@@@part "partName"]` and if the
-block has the label `part=partName`:
+using the label `file`. OCaml files can be sliced using named blocks:
+
+    ```ocaml
```

Comment:
I think you don't want this indentend as here we're showing a fragment of ocaml code.

If we indent it like this, it will be displayed as a code block containing the full markdown block, including the delimiters.

To clarify I think what we want here is this:
```ocaml
(* $MDX part-begin=partName *)
let meaning_of_life () =
  print_endline "42"
(* $MDX part-end *)
```

but we'd currently get this:

    ```ocaml
    (* $MDX part-begin=partName *)
    let meaning_of_life () =
      print_endline "42"
    (* $MDX part-end *)
    ```
    ```

Reply:
You're completely correct. I somehow didn't notice there's two layers of code blocks in the README, but it makes sense now that we want to show people how to write the code in Markdown yet show them how to mark up their OCaml code in OCaml syntax. Pushed a fix.